server/src/main/java/com/objectcomputing/checkins/services/guild/GuildServicesImpl.java

```diff
@@ -56,10 +59,25 @@ public void setEmailSender (EmailSender emailSender){
         this.emailSender = emailSender;
     }
 
+    public boolean validateLink (String link ) {
+        String regex = "(https://)?(www.)?compass\\.objectcomputing\\.com/S*";
```

Comment:
We should not be restricting this to an OCI site. This application is intended to be more broadly useful.

Reply:
I changed the URL validation to more broadly parse URL content.